db/organizations/cookieyes.eno

```diff
@@ -0,0 +1,6 @@
+name: CookieYes
+website_url: https://www.cookieyes.com/
+privacy_policy_url: https://www.cookieyes.com/privacy-policy/
+privacy_contact: support@cookieyes.com
+country: UK
```

Comment:
Took me a bit to understand why this broke the test. Unfortunately, it is not easy to see in the Github output either, but it expects UK to be GB. :-)